spec/daisybill_api/models/service_line_item_spec.rb

```diff
@@ -4,8 +4,8 @@
   it_behaves_like DaisybillApi::Ext::Attributes, :id, :units, :supplemental_data, :procedure_code,
     :diagnosis_code_1, :diagnosis_code_2, :diagnosis_code_3, :diagnosis_code_4, :charge_cents,
     :expected_payment_cents, :allowed_cents, :balance_due_cents, :write_off_cents, :created_at,
-    :updated_at, :custom_unit_charge_cents, modifier_codes: :collection
+    :updated_at, :custom_unit_charge_cents, :_destroy, modifier_codes: :collection
```

Comment:
Line is too long. [82/80]<br>Align the parameters of a method call if they span more than one line.